lavalink/Client.py

```diff
@@ -31,18 +25,16 @@ def __init__(self, bot, log_level=logging.INFO, loop=asyncio.get_event_loop(), h
         self.bot = bot
         self.bot.add_listener(self.on_socket_response)
 
+        self.user_id = self.bot.user.id

```

Comment:
This won't work if cogs are loaded before the bot logs in - will throw a `cannot get id from NoneType` error, needs a `wait_until_ready()` call